packages/botonic-nlp/src/tasks/ner/models/bilstm-model.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  input,
+  layers,
+  LayersModel,
+  model,
+  RNN,
+  SymbolicTensor,
+  Tensor2D,
+  train,
+} from '@tensorflow/tfjs-node'
+
+export function createBiLstmModel(
+  maxLength: number,
+  entities: string[],
+  embeddingsMatrix: Tensor2D
+): LayersModel {
+  const inputs = input({ name: 'InputLayer', shape: [maxLength] })
+
+  const embeddingsLayer = layers.embedding({
+    name: 'EmbeddingsLayer',
+    inputDim: embeddingsMatrix.shape[0] as number,
+    outputDim: embeddingsMatrix.shape[1] as number,
+    inputLength: maxLength,
+    weights: [embeddingsMatrix],
+    trainable: true,
+  })
+
+  const dropout = layers.dropout({ name: 'DropoutLayer', rate: 0.1 })
+
+  const bidirectionalLSTM = layers.bidirectional({
+    name: 'BidirectionalLayer',
+    layer: layers.lstm({
```

Comment:
let lstm parameters to be configurable from outside `createBiLstmModel(number, entities, embeddingsMatrix, config?:lstmConfig)`